tutorials/compiler-explorer-deployment.md

```diff
@@ -0,0 +1,219 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# Tutorial: Compiler Explorer Deployment
+
+Full instructins are available at [compiler-explorer: Adding a new library](https://github.com/compiler-explorer/compiler-explorer/blob/main/docs/AddingALibrary.md). This page is a Beman cheatsheet.
+
+## Integrating a header-only library
+
+### Step by step example: `beman.optional`
+
+1. Clone Compiler Explorer repos: <https://github.com/compiler-explorer/infra> and <https://github.com/compiler-explorer/compiler-explorer>.
```

Comment:
I feel it would be easier to have the commands in bash as well, I think everything should be copy paste to be able to quickly follow the content.

Reply:
done